Dockerfile

```diff
@@ -1,23 +1,28 @@
-FROM ubuntu:18.04
-MAINTAINER DevOps "devops@medicmobile.org"
+FROM node:16-slim
```

Comment:
The ultimate container size is `~1GB`.  I spent a considerable amount of time trying to get things to work with an `alpine` container, but never could get it to work with the way that our config tests run chromium via the test harness and Puppeteer.  

Also I figured that for a VSCode Dev Container setup Debian might be a better environment than Alpine (more familiar to someone using a terminal and possibly more compatible [Dev Container Features](https://containers.dev/features)).